doc/contributing.rst

```diff
@@ -1,221 +1,760 @@
-:orphan:
+.. contents:: Page contents
+   :local:
+   :depth: 2
 
-.. include:: links.inc
+.. _contributing:
 
-.. _contribute_to_mne:
+.. include:: ../CONTRIBUTING.rst
 
-How to contribute to MNE-python
-===============================
 
-.. contents:: Contents
-   :local:
-   :depth: 1
+Overview of contribution process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We are open to many types of contributions, from bugfixes to functionality
-enhancements. mne-python_ is meant to be maintained by a community of labs,
-and as such, we seek enhancements that will likely benefit a large proportion
-of the users who use the package.
+In general you'll be working with three different copies of the MNE-Python
+codebase: the official copy at https://github.com/mne-tools/mne-python (usually
+called "upstream"), your `fork`_ of the upstream repository (similar URL, but
```

Comment:
I would say "your remote fork" and earlier "the official remote copy at" -- it's perhaps redundant but it makes a distinction between what is in the cloud and what is no your HDD. It also eventually makes sense once people start thinking about `git remote ...` commands.